{"name": "test_order_details_modal[chrome]", "status": "broken", "statusDetails": {"message": "RuntimeError: Не удалось найти номер заказа: Message: Не дождались отображения номера заказа\nStacktrace:\n\tGetHandleVerifier [0x0xd6f173+62931]\n\tGetHandleVerifier [0x0xd6f1b4+62996]\n\t(No symbol) [0x0xb81053]\n\t(No symbol) [0x0xbc97ae]\n\t(No symbol) [0x0xbc9b4b]\n\t(No symbol) [0x0xc121f2]\n\t(No symbol) [0x0xbee4f4]\n\t(No symbol) [0x0xc0f9e3]\n\t(No symbol) [0x0xbee2a6]\n\t(No symbol) [0x0xbbd5f0]\n\t(No symbol) [0x0xbbe464]\n\tGetHandleVerifier [0x0xfc3463+2504899]\n\tGetHandleVerifier [0x0xfbe892+2485490]\n\tGetHandleVerifier [0x0xd9590a+220522]\n\tGetHandleVerifier [0x0xd86388+157672]\n\tGetHandleVerifier [0x0xd8cb6d+184269]\n\tGetHandleVerifier [0x0xd77158+95672]\n\tGetHandleVerifier [0x0xd77300+96096]\n\tGetHandleVerifier [0x0xd621aa+9738]\n\tBaseThreadInitThunk [0x0x75f15d49+25]\n\tRtlInitializeExceptionChain [0x0x77abd03b+107]\n\tRtlGetAppContainerNamedObjectPath [0x0x77abcfc1+561]", "trace": "self = <pages.order_feed_page.OrderFeedPage object at 0x000001EEE78D2120>\n\n    @allure.step(\"Получить номер первого заказа\")\n    def get_first_order_number(self) -> str:\n        \"\"\"Возвращает номер первого заказа в ленте с ожиданием\"\"\"\n        try:\n>           element = self._wait_until(\n                EC.visibility_of_element_located(self._locators.FIRST_ORDER_NUMBER),\n                message=\"Не дождались отображения номера заказа\"\n            )\n\npages\\order_feed_page.py:45: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.order_feed_page.OrderFeedPage object at 0x000001EEE78D2120>\ncondition = <function visibility_of_element_located.<locals>._predicate at 0x000001EEE8DAEC00>, timeout = None, message = 'Не дождались отображения номера заказа'\n\n    def _wait_until(self,\n                    condition: Callable[[WebDriver], Any],\n                    timeout: Union[int, float, None] = None,\n                    message: str = \"\") -> Any:\n        \"\"\"\n        Абстрактное ожидание условия\n        :param condition: Ожидаемое условие\n        :param timeout: Время ожидания\n        :param message: Сообщение об ошибке\n        \"\"\"\n        try:\n            wait_timeout = float(timeout) if timeout is not None else self.timeout\n>           return WebDriverWait(self.driver, wait_timeout).until(condition, message)\n\npages\\base_page.py:48: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"213b37c9a84c964e73e8e12f1ef25846\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x000001EEE8DAEC00>, message = 'Не дождались отображения номера заказа'\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Не дождались отображения номера заказа\nE       Stacktrace:\nE       \tGetHandleVerifier [0x0xd6f173+62931]\nE       \tGetHandleVerifier [0x0xd6f1b4+62996]\nE       \t(No symbol) [0x0xb81053]\nE       \t(No symbol) [0x0xbc97ae]\nE       \t(No symbol) [0x0xbc9b4b]\nE       \t(No symbol) [0x0xc121f2]\nE       \t(No symbol) [0x0xbee4f4]\nE       \t(No symbol) [0x0xc0f9e3]\nE       \t(No symbol) [0x0xbee2a6]\nE       \t(No symbol) [0x0xbbd5f0]\nE       \t(No symbol) [0x0xbbe464]\nE       \tGetHandleVerifier [0x0xfc3463+2504899]\nE       \tGetHandleVerifier [0x0xfbe892+2485490]\nE       \tGetHandleVerifier [0x0xd9590a+220522]\nE       \tGetHandleVerifier [0x0xd86388+157672]\nE       \tGetHandleVerifier [0x0xd8cb6d+184269]\nE       \tGetHandleVerifier [0x0xd77158+95672]\nE       \tGetHandleVerifier [0x0xd77300+96096]\nE       \tGetHandleVerifier [0x0xd621aa+9738]\nE       \tBaseThreadInitThunk [0x0x75f15d49+25]\nE       \tRtlInitializeExceptionChain [0x0x77abd03b+107]\nE       \tRtlGetAppContainerNamedObjectPath [0x0x77abcfc1+561]\n\nvenv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:101: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.test_order_feed.TestOrderFeed object at 0x000001EEE77CB4D0>, driver = <undetected_chromedriver.Chrome (session=\"213b37c9a84c964e73e8e12f1ef25846\")>\n\n    @allure.story(\"Открытие модального окна с деталями заказа\")\n    def test_order_details_modal(self, driver):\n        order_feed = OrderFeedPage(driver)\n        order_feed.open()\n    \n>       order_number = order_feed.get_first_order_number()\n\ntests\\test_order_feed.py:16: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.order_feed_page.OrderFeedPage object at 0x000001EEE78D2120>\n\n    @allure.step(\"Получить номер первого заказа\")\n    def get_first_order_number(self) -> str:\n        \"\"\"Возвращает номер первого заказа в ленте с ожиданием\"\"\"\n        try:\n            element = self._wait_until(\n                EC.visibility_of_element_located(self._locators.FIRST_ORDER_NUMBER),\n                message=\"Не дождались отображения номера заказа\"\n            )\n            return element.text\n        except TimeoutException as e:\n>           raise RuntimeError(f\"Не удалось найти номер заказа: {str(e)}\")\nE           RuntimeError: Не удалось найти номер заказа: Message: Не дождались отображения номера заказа\nE           Stacktrace:\nE           \tGetHandleVerifier [0x0xd6f173+62931]\nE           \tGetHandleVerifier [0x0xd6f1b4+62996]\nE           \t(No symbol) [0x0xb81053]\nE           \t(No symbol) [0x0xbc97ae]\nE           \t(No symbol) [0x0xbc9b4b]\nE           \t(No symbol) [0x0xc121f2]\nE           \t(No symbol) [0x0xbee4f4]\nE           \t(No symbol) [0x0xc0f9e3]\nE           \t(No symbol) [0x0xbee2a6]\nE           \t(No symbol) [0x0xbbd5f0]\nE           \t(No symbol) [0x0xbbe464]\nE           \tGetHandleVerifier [0x0xfc3463+2504899]\nE           \tGetHandleVerifier [0x0xfbe892+2485490]\nE           \tGetHandleVerifier [0x0xd9590a+220522]\nE           \tGetHandleVerifier [0x0xd86388+157672]\nE           \tGetHandleVerifier [0x0xd8cb6d+184269]\nE           \tGetHandleVerifier [0x0xd77158+95672]\nE           \tGetHandleVerifier [0x0xd77300+96096]\nE           \tGetHandleVerifier [0x0xd621aa+9738]\nE           \tBaseThreadInitThunk [0x0x75f15d49+25]\nE           \tRtlInitializeExceptionChain [0x0x77abd03b+107]\nE           \tRtlGetAppContainerNamedObjectPath [0x0x77abcfc1+561]\n\npages\\order_feed_page.py:51: RuntimeError"}, "steps": [{"name": "Открыть ленту заказов", "status": "passed", "start": 1748820962012, "stop": 1748820963870}, {"name": "Получить номер первого заказа", "status": "broken", "statusDetails": {"message": "RuntimeError: Не удалось найти номер заказа: Message: Не дождались отображения номера заказа\nStacktrace:\n\tGetHandleVerifier [0x0xd6f173+62931]\n\tGetHandleVerifier [0x0xd6f1b4+62996]\n\t(No symbol) [0x0xb81053]\n\t(No symbol) [0x0xbc97ae]\n\t(No symbol) [0x0xbc9b4b]\n\t(No symbol) [0x0xc121f2]\n\t(No symbol) [0x0xbee4f4]\n\t(No symbol) [0x0xc0f9e3]\n\t(No symbol) [0x0xbee2a6]\n\t(No symbol) [0x0xbbd5f0]\n\t(No symbol) [0x0xbbe464]\n\tGetHandleVerifier [0x0xfc3463+2504899]\n\tGetHandleVerifier [0x0xfbe892+2485490]\n\tGetHandleVerifier [0x0xd9590a+220522]\n\tGetHandleVerifier [0x0xd86388+157672]\n\tGetHandleVerifier [0x0xd8cb6d+184269]\n\tGetHandleVerifier [0x0xd77158+95672]\n\tGetHandleVerifier [0x0xd77300+96096]\n\tGetHandleVerifier [0x0xd621aa+9738]\n\tBaseThreadInitThunk [0x0x75f15d49+25]\n\tRtlInitializeExceptionChain [0x0x77abd03b+107]\n\tRtlGetAppContainerNamedObjectPath [0x0x77abcfc1+561]\n\n", "trace": "  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\pages\\order_feed_page.py\", line 51, in get_first_order_number\n    raise RuntimeError(f\"Не удалось найти номер заказа: {str(e)}\")\n"}, "start": 1748820963870, "stop": 1748820984955}], "parameters": [{"name": "driver", "value": "'chrome'"}], "start": 1748820962012, "stop": 1748820984957, "uuid": "de910f4e-20c3-4c7d-9314-3f6519014c2e", "historyId": "d36b86f1b9cb54035eeee63ddf9d5d6e", "testCaseId": "917601cac8d54cc647586f9f5c7c5d6d", "fullName": "tests.test_order_feed.TestOrderFeed#test_order_details_modal", "labels": [{"name": "feature", "value": "Лента заказов"}, {"name": "story", "value": "Открытие модального окна с деталями заказа"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_feed"}, {"name": "subSuite", "value": "TestOrderFeed"}, {"name": "host", "value": "Gemini_78"}, {"name": "thread", "value": "8968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_feed"}]}